validation/checks/ark_check_test.go

```diff
@@ -1,8 +1,7 @@
 //go:build unit
 
-// Package checks consists of individual validation checks.
-//
-// This file checks Archival Resource Keys (ARKs).
+// This file tests if the ARKCheck properly functions
```

Comment:
Period at end. And, not that the original comment was much more descriptive, but maybe explain more than "properly functions"(?) What does it do?

Reply:
Oh, nevermind on that latter part. I think we can be less descriptive for tests vs. actual code.